plugins/CoreAdminHome/Commands/DeleteConfig.php

```diff
@@ -0,0 +1,113 @@
+<?php declare(strict_types=1);
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Commands;
+
+use Piwik\Config;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\CoreAdminHome\Commands\DeleteConfig\ConfigDeletingManipulation;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DeleteConfig extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('config:delete');
+        $this->setDescription('Delete one or more config settings in the file config/config.ini.php');
+        $this->addArgument('assignment', InputArgument::OPTIONAL | InputArgument::IS_ARRAY, "List of config setting assignments, eg, section, section.config_setting_key or section.config_setting_key[position]");
+        $this->addOption('section', null, InputOption::VALUE_REQUIRED, 'The section the INI config setting belongs to.');
+        $this->addOption('key', null, InputOption::VALUE_REQUIRED, 'The name of the INI config setting.');
+        $this->setHelp("This command can be used to remove INI config settings on a Piwik instance.
+
+You can remove config values two ways, via --section, --key or by command arguments.
+
+To use --section, --key, simply supply those options. You can remove one whole section or one key inside the section.
+
+To use arguments, supply one or more arguments in the following format:
+
+Remove section:
+$ ./console config:delete 'section'
```

Comment:
@javier-de-juan  do you need the feature to remove an entire section? Cause if used wrongly it could do a lot of harm and break Matomo (eg if someone removes `General` or any other sections and forgets to pass a `key`, then Matomo be broken probably). If you need this feature, we'd probably need another flag like `only-section` just to be sure this is wanted

Reply:
Hi @tsteur! In my case it's not needed but I tried to think in all cases. I can remove this feature and keep it simple. I'll do it ASAP, today will be done ;)

Reply:
I was thinking about this feature and, in my opinion, it could be useful. Finally I decided to put a confirm question to avoid undesired behaviour. What do you think???

Reply:
Not sure where it would be useful? Not sure there is any section that can be removed without breaking anything actually.

Reply:
Ok @tsteur The feature was also removed. Actually the command will only remove settings in sections 👍